lib/presentation/mixins/media_picker_mixin.dart

```diff
@@ -218,10 +219,66 @@ mixin MediaPickerMixin on CommonMediaPickerMixin {
                               }
                               Navigator.of(context).pop();
                             },
-                            child: SvgPicture.asset(
-                              ImagePaths.icSend,
-                              width: 40,
-                              height: 40,
+                            child: SizedBox(
+                              width: 48,
```

Comment:
Shouldn't those sizes be in a `media_picker_style` file or something like that ?

Reply:
Done